test/integration/StreamrClient.spec.js

```diff
@@ -22,6 +24,16 @@ describe('StreamrClient', function () {
         opts = Object.assign({}, defaultOptions, opts)
         return new StreamrClient(opts)
     }
+    
+    before(() => {
+        return fetch('http://localhost:8081')
+            .catch((e) => {
+                if (e.errno === 'ENOTFOUND') {
+                    throw new Error('Integration testing requires the api and http-api ("entire stack") to be run in the background. ' +
```

Comment:
- Use correct project names: "api and http-api" => "engine-and-editor and data-api"
- Could also check that http://localhost:8890 (data-api) responds